### PR TITLE
Support i8 type

### PR DESCRIPTION
**Pull request by nandorKollar**
Problem

Compilation failure with i8 type

Solution

Introduce i8 type identifier as an alias for byte type.

Result

### CODE REVIEW

**Comment by mosesn** [comment]
this seems reasonable to me, thanks for the patch!!  how would you feel about adding something to one of the goldfile tests that validates that this works?

**Comment by nandorKollar** [comment]
@mosesn sure I can! Just committed a change related to gold files, did you think of these tests?

**Comment by mosesn** [comment]
LGTM!  @yufangong do you want to take a look here before we pull this in?

**Comment by yufangong** [comment]
this will fix https://github.com/twitter/scrooge/issues/309

File: scrooge-generator-tests/src/test/resources/gold_file_output_cocoa/TFNTwitterThriftGoldRequest.h [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /**
  * Generated by Scrooge
- *   version: ?
- *   rev: ?
- *   built at: ?
+ *   version: 19.4.0-SNAPSHOT
```
Comment (yufangong) [inline scrooge-generator-tests/src/test/resources/gold_file_output_cocoa/TFNTwitterThriftGoldRequest.h]: Can we revert these?

File: scrooge-generator-tests/src/test/scala/com/twitter/scrooge/goldfile/GoldFileTest.scala [hunk 2]
```diff
@@ -62,7 +62,7 @@ abstract class GoldFileTest extends FunSuite with BeforeAndAfterAll {
 
   protected def experimentFlags: Seq[String] = Seq.empty
   protected def language: String
-  protected def deleteTempFiles: Boolean = true
+  protected def deleteTempFiles: Boolean = false
```
Comment (yufangong) [inline scrooge-generator-tests/src/test/scala/com/twitter/scrooge/goldfile/GoldFileTest.scala]: revert?

**Comment by nandorKollar** [comment]
Ouch, I thought I reverted these intermediate changes before submitting the PR, but looks like I didn't. @yufangong thanks for the review, addressed the comments.

**Comment by yufangong** [comment]
Hi @nandorKollar, your PR has been merged here https://github.com/twitter/scrooge/commit/03d84aea40c9edf87e8c69090c5077b8779fa655. Thanks so much for your contribution.

**Pull request closed without merging**